client/my-sites/sidebar-unified/index.jsx

```diff
@@ -62,6 +69,10 @@ export const MySitesSidebarUnified = ( { path } ) => {
 	// link on new tab in order to avoid Happy Chat session disconnection.
 	// We return a bool that shows if the logic should terminate here.
 	const continueInCalypso = ( url, event ) => {
+		if ( isJetpackNonAtomicSite ) {
```

Comment:
I think there is no need of adding this condition here. Even if the site is `Jetpack` and the user has a happychat active, we should open external links on a new tab. Removing this condition will result in removing all code changes to this file too.

Reply:
This is to avoid showing the warning dialog so we maintain the existing behavior in the old navigation for Jetpack sites.

Reply:
But this will break Happychat, right? It seems that fixing this flow is simpler than maintaining the old behavior.

Reply:
In https://github.com/Automattic/wp-calypso/pull/50444/commits/3c8c6d27ef3957a03beea37633a33e0fbeac7c09 I added a comment explaining the reasoning behind this check, and also placed it inside the `isExternal` check, so it's a bit clearer.